pkg/universe.dagger.io/alpha/uffizzi/run.cue

```diff
@@ -0,0 +1,75 @@
+package uffizzi
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+	"universe.dagger.io/docker"
+	"universe.dagger.io/bash"
+)
+
+#Run: {
+	source: dagger.#FS
+
+	uffizzi_user:     string
+	uffizzi_password: dagger.#Secret
+	uffizzi_project:  *null | string
+	uffizzi_server:   string
+
+	docker_compose_file: *null | string
+	deployment_id:       *null | string
```

Comment:
Same here, why `null` instead of optional?

Reply:
Updated to optional field with values which make more sense. This was an oversight on my part. Thanks for checking.